src/main/kotlin/eu/kevin/api/models/ErrorCode.kt

```diff
@@ -7,11 +7,54 @@ import kotlinx.serialization.Serializable
 @Serializable(with = ErrorCodeSerializer::class)
 enum class ErrorCode(val code: Int) {
     @SerialName("10000") SYSTEM_ERROR(10000),
+    @SerialName("10001") CANT_GET_DATA(10001),
+    @SerialName("10005") INVALID_OR_EXPIRED_TOKEN(10005),
+    @SerialName("10021") INVALID_PAYMENT_DATA(10021),
+    @SerialName("10024") LOGIN_WAS_NOT_STARTED(10024),
+    @SerialName("10030") CANT_PROCESS_PAYMENT(10030),
+    @SerialName("10032") CONSENT_NOT_CREATED(10032),
+    @SerialName("10035") LOGIN_ALREADY_FINISHED(10035),
+    @SerialName("11000") INVALID_PSU_IP_ADDRESS(11000),
+
+    @SerialName("20001") CURRENT_BANK_ERROR(20001),
+    @SerialName("20002") OAUTH_TOKEN_OR_CODE_NOT_VALID_ANYMORE(20002),
+    @SerialName("20010") PAYMENT_PRODUCT_NOT_SUPPORTED_BY_BANK(20010),
+    @SerialName("20013") INCORRECT_BANK_ID(20013),
+    @SerialName("20014") NOT_SUPPORTED_BANK_SECURECUSTOMERAUTH_METHOD(20014),
```

Comment:
is this name ok? I mean merged `SECURECUSTOMERAUTH`

Reply:
+1

Reply:
when directly taken from documentation:
20014: Not supported bank secureCustomerAuth method.